frontend/app/services/TouchpointBackend.scala

```diff
@@ -70,6 +72,15 @@ case class TouchpointBackend(
       _ <- subscriptionService.downgradeSubscription(member, FriendTierPlan)
     } yield {
       memberRepository.metrics.putDowngrade(member.tier)
+      track(
+        MemberActivity(
+          "downgradeMembership",
+          MemberData(
+          member.salesforceContactId,
```

Comment:
I did start with this but in some cases member just references the common trait MemberId which has just salesforce details. So other information has to come from Identity or the form.